src/prefect/tasks/docker/__init__.py

```diff
@@ -0,0 +1,16 @@
+from prefect.tasks.docker.images import (
```

Comment:
Could you add a docstring to the top of this file explaining any requirements for a docker image if it is going to be able to run these tasks?

Reply:
Like a requirement for an image on which these tasks will run? (e.g. running your flow with these tasks on a python:3.6 container, etc...)

Reply:
Yea exactly; do users need to be aware of any additional installations that need to be in their docker image for these tasks to run properly?